src/pages/MonthlyGood/HowItWorksExperiment.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="how-works-section-wrapper">
+		<div class="row">
+			<h1 class="column small-12 text-center impact-text">
+				How it works
+			</h1>
+
+			<div class="small-12 large-4 column step-wrapper text-center">
+				<icon-auto-deposit-alternate class="mg-image" />
+				<h4>Contribute automatically</h4>
+				<div class="description-text">
+					<p>Sign up once by choosing a monthly amount that works for you. Cancel anytime.</p>
+				</div>
+			</div>
+			<div class="small-12 large-4 column step-wrapper text-center">
+				<icon-lend class="mg-image" />
+				<h4>Help entrepreneurs in need</h4>
+				<div class="description-text">
+					<!-- eslint-disable-next-line max-len -->
+					<p>Kiva will automatically lend 100% of your funds to borrowers who need help reaching their goals. This could be anyone on Kiva, from farmers to students, artisans to shopkeepers.</p>
+				</div>
+			</div>
+			<div class="small-12 large-4 column step-wrapper text-center">
+				<icon-updates-alternate class="mg-image" />
+				<h4>Receive inspiring stories</h4>
+				<div class="description-text">
+					<p>We’ll send you an update once a month, including who you supported and why it mattered.</p>
+				</div>
+			</div>
+			<div class="small-12 column text-center learn-more-wrapper">
+				<p class="learn-more-text">
+					<!-- eslint-disable-next-line max-len -->
+					As you’re repaid, your funds will be automatically relent to borrowers in need, multiplying your impact
+				</p>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import IconAutoDepositAlternate from '@/assets/icons/inline/auto-deposit-alternate.svg';
+import IconLend from '@/assets/icons/inline/lend.svg';
+import IconUpdatesAlternate from '@/assets/icons/inline/updates-alternate.svg';
+
+export default {
+	components: {
+		IconAutoDepositAlternate,
+		IconLend,
+		IconUpdatesAlternate,
+	},
+};
+
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.how-works-section-wrapper {
+	@include breakpoint(medium only) {
+		padding-top: 12rem;
+	}
+
+	@include breakpoint(small only) {
+		padding-top: 14rem;
+	}
```

Comment:
When using breakpoints to change the padding, it's better to use a small-screen first approach, in this case that could look like:
```
.how-works-section-wrapper {
    padding-top: 14rem;
    @include breakpoint(medium) {
        padding-top: 12rem;
    }
    @include breakpoint(large) {
        padding-top: 0; // or whatever you want this to be
    }
```

Reply:
@emuvente Thanks for the comments. I'm new to front-end dev so i'm figuring it out as I go 😄